nautobot/ui/src/components/common/paginator.jsx

```diff
@@ -29,14 +29,20 @@ export default function Paginator({ url, data_count, page_size, active_page }) {
   }
 
   for (let i = 0; i < num_pages; i++) {
+    /*
     if (i === active_page) {
       pages.push(<Pagination.Item active key={i} href={list_url + `?limit=${page_size}&offset=${page_size * i}`}>{i + 1}</Pagination.Item>)
     } else {
       pages.push(<Pagination.Item key={i} href={list_url + `?limit=${page_size}&offset=${page_size * i}`}>{i + 1}</Pagination.Item>)
     }
+    */
```

Comment:
Probably should add TODOs to this and other commented-out code blocks in this PR.